.github/workflows/flutter.yml

```diff
@@ -75,10 +75,10 @@ jobs:
         env:
           TOTAL: ${{ steps.analysis.outputs.total }}
           TOTAL_MAX: ${{ steps.analysis.outputs.total_max }}
-        # TODO: Once 1.0.0 lands, change to 100 
+        # TODO: Once 4.0.0 lands, change to 100 
         run: |
           PERCENTAGE=$(( $TOTAL * 100 / $TOTAL_MAX ))
-          if (( $PERCENTAGE < 95 ))
+          if (( $PERCENTAGE < 90 ))
```

Comment:
Is there a follow up PR fixing? Or maybe we raise an issue to get this to 100 before GA?

Reply:
lets track a separate issue